thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/ThirdEyeDashboardConfiguration.java

```diff
@@ -11,7 +11,9 @@
   String rcaConfigPath;
   int rcaParallelism;
   List<String> rcaFormatters;
-  String rcaDefaultFramework;
```

Comment:
It's mostly a limitation of how we load configs. If a property is not in the config file, it errors out. We could could change that and make RCA (and other stuff) optional